README.md

```diff
@@ -45,7 +45,7 @@ server:
 
 | kured  | kubectl | k8s.io/client-go | k8s.io/apimachinery | expected kubernetes compatibility |
 |--------|---------|------------------|---------------------|-----------------------------------|
-| master | 1.18.8  | v0.18.8          | v0.18.8             | 1.17.x, 1.18.x, 1.19.x            |
+| master | 1.19.4  | v0.19.4          | v0.19.4             | 1.17.x, 1.18.x, 1.19.x            |
```

Comment:
There is no `kindest/node` image for 1.20 yet - maybe unofficially somewhere? But we can start testing locally I guess.

Reply:
Just pointing this out here as to why I didn't update the "expected k8s compatibility" section. We can do that in the next PR.

Reply:
Yeah we can probably build one. Or maybe we should wait for the official release before working on 1.20? We are so much ahead, it's impressive :)

Reply:
Shouldn't we edit this to 1.17.x,1.18.x,1.19.x?